### DISCUSSION TITLE
Remove ooc Queuing out of WvW

### DISCUSSION THREAD

**Discussion by Bumpershoot** [bennieboj/BlishHud-Mounts#45] [General]
Unsure if you've encountered this but it seems unintended, sometimes in WvW you'll be standing in-combat and suddenly you are mounted because you were OoC for a small tick of server time, but you are still in-combat and unable to say, waypoint. This can create unfair scenarios in a pvp gamemode.

**Comment by bennieboj** [comment]
I have not experienced this. Can you provide a video?

Also, what are you requesting? To completely remove ooc queuing in wvw?

**Comment by bennieboj** [comment]
Not sure if this helps, but I added some options to disable mount queuing.
https://discord.com/channels/531175899588984842/1136363059674357810
Implemented in 1.4.0, currently in beta

I didn't receive any feedback so for now I'm going to close it.
If you still have this issue with the latest version, please make an issue with more details.
Thanks!
